storage-bigtable/src/bigtable.rs

```diff
@@ -654,6 +654,7 @@ mod tests {
                 pre_balances: vec![43, 0, 1],
                 post_balances: vec![0, 42, 1],
                 inner_instructions: Some(vec![]),
+                log_messages: Some(vec![]),
             }),
         };
         let block = ConfirmedBlock {
```

Comment:
Looks like you need to set `log_messages: None` for the expected bincode block ~line 705 below

Reply:
Ah, thank you!